systemd/systemd.go

```diff
@@ -142,6 +143,11 @@ func NewCollector(logger log.Logger) (*Collector, error) {
 		"Last time the unit transitioned out of the inactive state",
 		[]string{"name", "type"}, nil,
 	)
+	unitMemoryCurrentDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "unit_memory_current"),
```

Comment:
not exactly sure about the naming here, I've stolen most of the code from the `tasks` metric, but it really is only applicable to services, not units in general, so should probably be `service_memory_current`?